test/integration/dss/test_dss_cli.py

```diff
@@ -31,24 +33,32 @@ def test_get_files_cli(self):
         filename = "SRR2967608_1.fastq.gz"
         dirpath = os.path.join(TEST_DIR, "res", "bundle")
         file_path = os.path.join(dirpath, filename)
-
+        dest_dir = tempfile.mkdtemp(dir=os.getcwd(), prefix="cli-test-", suffix=".tmp")
         replica = "aws"
         staging_bucket = "org-humancellatlas-dss-cli-test"
-        client = hca.dss.DSSClient()
-        response = client.upload(src_dir=dirpath, replica=replica, staging_bucket=staging_bucket)
 
-        for f in response["files"]:
+        upload_args = ['dss', 'upload', '--src-dir', dirpath, '--replica', replica, '--staging-bucket', staging_bucket]
+        with CapturingIO('stdout') as stdout_upload:
+            hca.cli.main(args=upload_args)
+        upload_res = json.loads(stdout_upload.captured())
+        for f in upload_res['files']:
             print(f)
             if f["name"] == filename:
                 file_uuid = f['uuid']
                 break
 
-        res = client.get_file(uuid=file_uuid, replica=replica)
+        download_args = ['dss', 'download', '--bundle-uuid', upload_res['bundle_uuid'], '--replica', replica,
+                         '--dest-name', dest_dir]
+        with CapturingIO('stdout') as stdout_download:
+            hca.cli.main(args=download_args)
 
-        self.assertIsInstance(res, bytes)
+        with open(os.path.join(dest_dir, filename), 'rb') as download_data:
+            download_content = download_data.read()
         with open(file_path, "rb") as bytes_fh:
             file_content = bytes_fh.read()
-            self.assertEqual(file_content, res)
+            self.assertEqual(file_content, download_content)
+
+        shutil.rmtree(dest_dir)
```

Comment:
As it stands now, this temp directory will not be cleaned-up if the test execution does not reach this point.
I propose putting the test body in a `try` block with the `finally` clause containing the removal of the directory.

Reply:
good catch, ill implement that now